src/states/elder/mod.rs

```diff
@@ -1830,8 +1820,14 @@ impl Elder {
         self.next_relocation_dst = dst;
     }
 
-    pub fn received_section_info_ack(&self) -> bool {
-        self.received_section_info_ack
+    pub fn get_version(&self) -> u64 {
+        let mut versions = self.chain.get_versions();
+        if let Some(version) = versions.next() {
+            assert!(versions.next().is_none());
```

Comment:
I am not clear why this would have an assert here.
it seem that if there is more than 2 sections, this would not be true.

I think you can probably already access the chain function without adding a new function here (chain accessor exists with mock_base enabled).

Reply:
I am not aware the whole chain is accessable.
Thought we prefer only access the scope of info that when we need to.

Will change to use the `chain` accessor` then.

Reply:
Existing code:
```
#[cfg(feature = "mock_base")]
impl Node {
    /// Returns the chain for this node.
    pub fn chain(&self) -> Option<&Chain> {
        self.machine.current().chain()
    }
```